modules/pages/client/components/Volunteering.component.js

```diff
@@ -67,7 +67,7 @@ export default function Volunteering() {
 
             <li>
               <strong><em>TEST THE WEBSITE!</em></strong>
-              <p>Use the website: travel and host! Test it out with different mobile devices, browsers, and screen sizes. Report bugs by opening <a href="https://github.com/Trustroots/trustroots/issues/new">an issue on GitHub</a> or use the <a ui-sref="contact">contact</a> form. </p>
+              <p>Use the website: travel and host! Test it out with different mobile devices, browsers, and screen sizes. Report bugs by opening <a href="https://github.com/Trustroots/trustroots/issues/new">an issue on GitHub</a> or use the <a href="/contact">contact</a> form. </p>
```

Comment:
@guaka so just for the record that this for now only from React components. ;-)

@mrkvon i18n PR will need to be rebased with this commit.